lib/modules/manager/gradle/extract/catalog.ts

```diff
@@ -14,6 +15,22 @@ import type {
   VersionPointer,
 } from '../types';
 
+function findVersionIndex(
+  content: string,
+  depName: string,
+  version: string
+): number {
+  const contents = content.replaceAll('\r\n', '\n');
```

Comment:
it works without this line on windows, but when i moved created a file in github and tried on a real repo, it failed cuz of this \r\n instead of \n. :/